jvm-packages/jenkins/Jenkinsfile-Release

```diff
@@ -91,8 +94,8 @@ try {
             // setup custom DetailsSummary
             DetailsSummary detailsSummary = new DetailsSummary()
             detailsSummary.setEntry(this, 'XGBoost Version', XGB_VERSION)
-            detailsSummary.setEntry(this, 'Deploy to Local Nexus', params.publishToLocalNexus ? 'Yes' : 'No')
-            detailsSummary.setEntry(this, 'Deploy to Public Nexus', params.publishToPublicNexus ? 'Yes' : 'No')
```

Comment:
Should it be exclusive choice? Or the other but not both?

Reply:
I'm not aware, that jenkins supports exclusive choices. We can do it using dropdown - Local Release, Public Release. Also we can probably restrict releases to `h2o3` branch only, WDYT?